scripts/common-service.Makefile

```diff
@@ -112,6 +112,7 @@ _run-test-dev: _check_venv_active
 		--failed-first \
 		--setup-show \
 		--pdb \
+		--asyncio-mode=auto \
```

Comment:
what benefit does this bring?

Reply:
latest pytest-aiohttp is now depending  on pytest-asyncio. If I don't put this in then the async tests are not running.